source/customization.rst

```diff
@@ -229,6 +232,33 @@ If you access the Dashboard, and it crashes, then you may have made a mistake
 in ``ood.rb`` file, whose code is run during the initialization of the Rails
 app.
 
+.. _set-upload-limits:
+
+Set Upload Limits
+-----------------
+
+By default, the file size upload limit is 10737420000 bytes (~10.7 GB).
+
+If you want set this to a lower value, set the ``FILE_UPLOAD_MAX`` configuration
+in the file apps' configuration file ``/etc/ood/config/apps/shell/env``.
+
+If you want to set it to a higher value, configure the files app as just described
+and also set ``nginx_file_upload_max`` in ``/etc/ood/config/nginx_stage.yml`` to
+the same number.
```

Comment:
For a higher value we only need to set `nginx_file_upload_max` since setting that also sets NGINX_FILE_UPLOAD_MAX env var that the Files app respects.